mappings/net/minecraft/entity/TrackedPosition.mapping

```diff
@@ -0,0 +1,21 @@
+CLASS net/minecraft/class_7422 net/minecraft/entity/TrackedPosition
```

Comment:
Why not `TrackedPos` to match `BlockPos` and `GlobalPos`?

Reply:
Hmm, maybe.

Reply:
You can't use this as `Vec3d`/`Vec3i`/whatever anyway, this is just a wrapper with network syncing utils.